synth.py

```diff
@@ -32,24 +32,20 @@
    version,
    generator_args={
      "grpc-service-config": f"grafeas/{version}/grafeas_grpc_service_config.json",
-     "package-name":f"@google-cloud/grafeas"
+     "package-name":"@google-cloud/grafeas"
      },
-     proto_path=f'/grafeas/{version}',
-     extra_proto_files=['google/cloud/common_resources.proto'],
+     proto_path=f'/grafeas/{version}'
      )
  s.copy(library, excludes=[
    "README.md",
-   "package.json",
-   "src/v1/gapic-grafeas-v1.ts",
```

Comment:
Do you know why it was listed as excluded? Is there any problem with generation? A little bit confused here.

Reply:
I checked the original conversion PR:
```
 s.copy(library, excludes=[
   "README.md",
   "package.json",
 -  "src/v1/grafeas_client.js",
 -  "test/gapic-v1.js",
 +  "src/v1/gapic-grafeas-v1.ts",
 +  "test/gapic-v1.ts",
   "protos/google/*"
 ])
```
I think for some reasons, the JS client lib skipped `src/v1/client` and `test/client.ts` so we did the same for TS client lib. But by removing these two files and rerun synthtool now, it works pretty well.